packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -144,6 +157,19 @@ const Messages: FunctionComponent<Props> = ({
   const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
     useTableSelect<Thread>(threads)
 
+  const [deletedThreads, setDeletedThreads] = useState<string[]>([])
+
+  useEffect(() => {
+    if (deletingState === ThreadDeletingState.Success) {
+      const timeout = setTimeout(() => {
+        closeModal()
```

Comment:
Maybe clear will sounds more generic from the view perspective?
```suggestion
        clearThreadDeletingState()
```

Reply:
Here `closeModal` changed to `hideDeleteModal`